### DISCUSSION TITLE
Fuzzy Searching with Regex

### DISCUSSION THREAD

**Discussion by MSSRPRAD** [rust-lang/regex#1045] [Q&A] [answered]
Does this crate support fuzzy searching on Regex pattern? If so can someone point me towards some examples/documentation for the same? `docs.rs` doesn't have any documentation for it... TIA

**Comment by BurntSushi** [comment, marked as answer]
Nope.

**Reply by MSSRPRAD** [reply]
Ah! Thanks for the very prompt reply. Do you know any other crate that does it? Would be a great help!

**Reply by BurntSushi** [reply]
I don't. Not in the context of general regex searching anyway. It's an uncommon (but not unheard of) feature to support in regex. You'll want to maybe look for levenshtein implementations.

**Reply by MSSRPRAD** [reply]
Thank you!
